x-pack/elastic-agent/docs/elastic-agent.asciidoc

```diff
@@ -1,78 +1,242 @@
-Agent's notes
-
-
-[[requirements]]
-Requirements
-
-The remote Kibana version of the Agent must be equal or higher than the Agent version, if this is not met
```

Comment:
@michalpristas I think the requirement is that the Agent must be equal or higher than the Agent minor version. I think we are still able to make patch releases for Agent without updating the whole stack. Can you confirm?

Reply:
we removed sending kbn-version as there was a check on kibana side which required it to be `==` when sent. not sure what the status of the check is atm cc @nchaulet